DigitalLearningSolutions.Data.Tests/DataServices/PasswordDataServiceTests.cs

```diff
@@ -48,112 +47,54 @@ public void SetPasswordByCandidateNumber_should_set_password_correctly()
         }
 
         [Test]
-        public async Task SetPasswordByEmailAsync_sets_password_for_matching_user()
+        public async Task SetPasswordByUserIdAsync_sets_password_for_matching_user()
         {
             using var transaction = new TransactionScope(TransactionScopeAsyncFlowOption.Enabled);
 
             // Given
-            var existingAdminUser = UserTestHelper.GetDefaultAdminUser();
+            var existingUser = UserTestHelper.GetDefaultUserAccount();
             var newPasswordHash = PasswordHashNotYetInDb;
 
             // When
-            await passwordDataService.SetPasswordByEmailAsync(existingAdminUser.EmailAddress!, newPasswordHash);
+            await passwordDataService.SetPasswordByUserIdAsync(existingUser.Id, newPasswordHash);
 
             // Then
-            userDataService.GetAdminUserById(existingAdminUser.Id)?.Password.Should()
+            userDataService.GetUserAccountById(existingUser.Id)?.PasswordHash.Should()

```

Comment:
Making this change turned up an error in the test suite for this particular file, although ironically with the _other_ instance of `?.`, rather than with the line above.

Good suggestion!